dearpygui/demo.py

```diff
@@ -2219,14 +2617,36 @@ def query(sender, app_data, user_data):
 
                         with dpg.plot(label="Drag Lines/Points", height=400, width=-1):
                             dpg.add_plot_legend()
-                            dpg.add_plot_axis(dpg.mvXAxis, label="x")
+                            x_axis = dpg.add_plot_axis(dpg.mvXAxis, label="x")
```

Comment:
`x_axis` is the only change on this line and it's not used. Can live without it.